app/views/layouts/_feedback.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="govuk-width-container">
+  <div class="gem-c-feedback">
```

Comment:
these class names are copied from the `govuk_publishing_components` - I'm not actually sure why they're called gem-c-... but thought I'd use the same names for clarity